generator/src/main/java/org/apache/kafka/message/MessageDataGenerator.java

```diff
@@ -1227,6 +1238,19 @@ private String fieldDefault(FieldSpec field) {
                 }
                 return field.defaultString() + "L";
             }
+        } else if (field.type() instanceof FieldType.UUIDFieldType) {
+            headerGenerator.addImport(MessageGenerator.UUID_CLASS);
+            if (field.defaultString().isEmpty()) {
+                return "org.apache.kafka.common.protocol.MessageUtil.ZERO_UUID";
```

Comment:
@cmccabe Maybe I missed the point; I thought this would set it to a "zero UUID" (previously, this was setting it to `null`).